seya/layers/recurrent.py

```diff
@@ -15,11 +15,23 @@ def _get_reversed_input(self, train=False):
 
 
 class Bidirectional(Recurrent):
-    def __init__(self, forward, backward, return_sequences=False,
-                 truncate_gradient=-1):
+    def __init__(self, forward=None, backward=None, return_sequences=False,
+                 truncate_gradient=-1, forward_conf=None, backward_conf=None):
+        assert forward is not None or forward_conf is not None, "Must provide a forward RNN or a forward configuration"
+        assert backward is not None or backward_conf is not None, "Must provide a backward RNN or a backward configuration"
         super(Bidirectional, self).__init__()
-        self.forward = forward
-        self.backward = backward
+        if forward is not None:
+            self.forward = forward
+        else:
+            # Must import inside the function, because in order to support loading
+            # we must import this module inside layer_utils... ugly
+            from keras.utils.layer_utils import container_from_config
```

Comment:
I can't as in keras/utils/layer_utils.py they import all possible containers at the top, so it will create a cyclic import. I think it's a bad design decision in keras, but won't fight for it right now.